base/src/com/google/idea/blaze/base/sync/aspects/strategy/AspectStrategyBazel.java

```diff
@@ -58,7 +58,10 @@ public Optional<File> aspectTemplateDirectory() {
   @VisibleForTesting
   public AspectStrategyBazel(BlazeVersionData versionData) {
     super(/* aspectSupportsDirectDepsTrimming= */ true);
-    if (versionData.bazelIsAtLeastVersion(6, 0, 0)) {
+    boolean useInjectedRepository = versionData.bazelIsAtLeastVersion(8, 0, 0);
+    if (useInjectedRepository) {
+      aspectFlag = "--aspects=@intellij_aspect//:intellij_info_bundled.bzl%intellij_info_aspect";
```

Comment:
why do we have to bring this to a single `@`?

Reply:
https://github.com/bazelbuild/bazel/issues/22691#issuecomment-2472920470